src/screen.cpp

```diff
@@ -368,20 +379,27 @@ void Screen::drawWidgets() {
             float bounds[4];
             nvgFontFace(mNVGContext, "sans");
             nvgFontSize(mNVGContext, 15.0f);
-            nvgTextAlign(mNVGContext, NVG_ALIGN_CENTER | NVG_ALIGN_TOP);
+            nvgTextAlign(mNVGContext, NVG_ALIGN_LEFT | NVG_ALIGN_TOP);
             nvgTextLineHeight(mNVGContext, 1.1f);
             Vector2i pos = widget->absolutePosition() +
                            Vector2i(widget->width() / 2, widget->height() + 10);
 
-            nvgTextBoxBounds(mNVGContext, pos.x(), pos.y(), tooltipWidth,
+			nvgTextBounds(mNVGContext, pos.x(), pos.y(),
                              widget->tooltip().c_str(), nullptr, bounds);
-
```

Comment:
ditto -- please do a pass over all the changes and replace tabs with four spaces ;)